010_getting-started/010_mapping-method/050_efi-bootloader.md

```diff
@@ -0,0 +1,32 @@
+# Using EFI bootloader to load the Valthrun driver
+
+**Pre-requisites**  
+   Before proceeding, there are several important steps and requirements to be aware of:
+
+   - **Kernel Anti-Cheats**  
+   Ensure all kernel anti-cheats are disabled before proceeding. Failing to do so may result in your system being flagged.
+
+   - **Secure Boot**  
```

Comment:
This is wrong, the driver works with secure boot :)  
Note:   
You have to enrol the hash manually